R/Reactant/build_tarballs.jl

```diff
@@ -25,10 +25,13 @@ echo GCC version: $(gcc --version)
 GCC_VERSION=$(gcc --version | head -1 | awk '{ print $3 }')
 GCC_MAJOR_VERSION=$(echo "${GCC_VERSION}" | cut -d. -f1)
 
-if [[ "${target}" == x86_64-apple-darwin* ]]; then
+if [[ "${target}" == *-darwin* ]]; then
     # Compiling LLVM components within XLA requires macOS SDK 10.14.
-    pushd $WORKSPACE/srcdir/MacOSX10.*.sdk
+    # But then we need 13.3 for another `std::hardware_destructive_interference_size`:
+    # <https://github.com/JuliaPackaging/Yggdrasil/pull/10313#issuecomment-2616125018>.
```

Comment:
@giordano we were just removing hw interference size: https://github.com/EnzymeAD/Reactant.jl/blob/fd60aad440379a9b04a0164f34cbd65cd679ed89/deps/ReactantExtra/WORKSPACE#L57